Client/Application/AppDelegate.swift

```diff
@@ -142,6 +142,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             self?.ratingPromptManager.updateData()
         }
 
+        DispatchQueue.global(qos: .default).async { [weak self] in
```

Comment:
nit: You don't need to specify .default, it's a default parameter so you can remove the qos parameter altogether.